dev/tracing_tests/test/image_cache_tracing_test.dart

```diff
@@ -0,0 +1,161 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+import 'dart:developer' as developer;
+import 'dart:io';
+import 'dart:isolate' as isolate;
+
+import 'package:flutter/painting.dart';
+
```

Comment:
nit: I believe this blank line shouldn't be here?

Reply:
Removed.